### PR TITLE
feat: ERC721 bridge and ERC721 native same contract address

### PR DESCRIPTION
**Pull request by cuonghx-dev**
This commit support deploying the same address feature:
- deploy same address contract ERC721
- deploy script
- Using id for salt when create new collection

### CODE REVIEW

File: contracts/tokens/ERC721NativeToken.sol [hunk 1]
```diff
@@ -0,0 +1,77 @@
+pragma solidity 0.7.5;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "../interfaces/IOwnable.sol";
+
+contract ERC721NativeToken is ERC721 {
```
Comment (kazkondo) [inline contracts/tokens/ERC721NativeToken.sol:6]: It should be ERC721BridgableToken ?

**Review by kazkondo — CHANGES_REQUESTED** [review]

File: contracts/tokens/ERC721NativeToken.sol (hunk 1 above)
Comment (cuonghx-dev) [reply contracts/tokens/ERC721NativeToken.sol:6]: - `ERC721NativeToken` deployed by the user 
- `ERC721BridgableToken` deployed by bridge after user bridging first NFT

Comment (kazkondo) [reply contracts/tokens/ERC721NativeToken.sol:6]: I mean that this token is the customized token by G.U.? Then we might be change the class name to more clear one. GUERC771NativaToken, etc. How do you think?

Comment (cuonghx-dev) [reply contracts/tokens/ERC721NativeToken.sol:6]: Kaz: `ERC721NativeToken` -> `GUERC721NativeToken`
(Sorry I deleted your comment by chance)

Comment (cuonghx-dev) [reply contracts/tokens/ERC721NativeToken.sol:6]: That's oke. But can I update it after merge because depend?

**Pull request merged**